File: src/main/java/com/manson/domain/fed76/AuthorResponse.java
package com.manson.domain.fed76;

import com.manson.domain.AbstractObject;
import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = false)
@ToString
public class AuthorResponse extends AbstractObject {
    private String name;
    private String logo;
    private String description;
    private String url;

}

File: src/main/kotlin/com/manson/domain/fed76/pricing/AuthorResponse.kt
package com.manson.domain.fed76.pricing

import com.manson.AbstractObject

open class AuthorResponse : AbstractObject() {
    var name: String = "FED76"
    var logo: String = "https://fed76.info/static/fed150.png"
    var description: String = "Battle-tested source of Fallout 76 information"
    var url: String = "https://fed76.info/"
    override fun toString(): String {
        return "AuthorResponse(name='$name', logo='$logo', description='$description', url='$url')"
    }


}